origami.json

```
@@ -1,6 +1,6 @@
 {
-	"description": "A layout component for internal products and tools",
-	"keywords": "layout, documentation",
+	"description": "Provides page layouts and typography as a starting point to create internal tools or products. Currently offers three layout types.",
```

Think the description probably will be forgotten about when we add or remove layouts
```suggestion
	"description": "Provides page layouts and typography as a starting point to create internal tools or products.",
```